readinglist/resource.py

```diff
@@ -76,6 +68,7 @@ class BaseResource(object):
     mapping = RessourceSchema()
     id_field = '_id'
     modified_field = 'last_modified'
+    schema_for = ('POST', 'PUT')
```

Comment:
I'm not a big fan of `schema_for`, but I cannot find any better names…

- `validation_methods`
- `schema_validation_methods` ?